projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/integration-card-header.tsx

```diff
@@ -38,7 +28,7 @@ const IntegrationCardHeader = ( {
 	onToggle,
 	cardData = {},
 	toggleTooltip,
-}: IntegrationCardHeaderProps ) => {
+}: IntegrationCardProps ) => {
```

Comment:
Again to ensure consistency between IntegrationCard and its child components, and because this component use a a nearly-complete subset of IntegrationCard props, I'm just proposing we re-use IntegrationCardProps. 

In this case, IntegrationCardProps has two properties that IngrationCardHeader does not need: `children?: ReactNode;` and `borderBottom?: boolean;`. But those are already optional, so we can safely reuse IntegrationCardProps here. 

I like the DRY-ness and type consistency here between parent/child components, but if anyone thinks we should still redefine a type just for this component, happy to tweak it.